### ISSUE TITLE
EU::CBuilder + EU::ParseXS + EU::MakeMaker test plan failures

### ISSUE THREAD

**Issue by ribasushi** [perl11/cperl#130]
Apologies if there is a ticket for this - I could not find it. When trying to compile and test cperl I reliably get the following failures. Let me know what other info you might need.

```
../dist/ExtUtils-CBuilder/t/00-have-compiler.t                  (Wstat: 6 Tests: 2 Failed: 0)
  Non-zero wait status: 6
  Parse errors: Bad plan.  You planned 7 tests but ran 2.
../dist/ExtUtils-ParseXS/t/001-basic.t                          (Wstat: 6 Tests: 3 Failed: 0)
  Non-zero wait status: 6
  Parse errors: Bad plan.  You planned 17 tests but ran 3.
../dist/ExtUtils-CBuilder/t/01-basic.t                          (Wstat: 6 Tests: 0 Failed: 0)
  Non-zero wait status: 6
  Parse errors: No plan found in TAP output
../dist/ExtUtils-CBuilder/t/02-link.t                           (Wstat: 6 Tests: 0 Failed: 0)
  Non-zero wait status: 6
  Parse errors: No plan found in TAP output
../dist/ExtUtils-ParseXS/t/002-more.t                           (Wstat: 6 Tests: 2 Failed: 0)
  Non-zero wait status: 6
  Parse errors: Bad plan.  You planned 29 tests but ran 2.
../dist/ExtUtils-ParseXS/t/003-usage.t                          (Wstat: 6 Tests: 2 Failed: 0)
  Non-zero wait status: 6
  Parse errors: Bad plan.  You planned 24 tests but ran 2.
../cpan/ExtUtils-MakeMaker/t/xs.t                               (Wstat: 6 Tests: 0 Failed: 0)
  Non-zero wait status: 6
  Parse errors: No plan found in TAP output
```

Everything else works with flying colors though. Thanks!

**Comment by ribasushi** [comment]
The above failures happen when doing: `perlbrew install --as 5.22.1_cperl -Duseithreads  -j10 -v /tmp/cperl-cperl-5.22.1.tar.gz`

**Comment by rurban** [comment]
Yes, those are known errors with 5.22.1, fixed with 5.22.2/master
Even worse is the Config bug, silently adding a \0 or " byte to all config values 
with a ", such as the sPRI\* format specifiers.

```
$ cperl5.22.1 -V:sPRIXU64
sPRIXU64='"lX""';
```
